utils/constants.js

```diff
@@ -2,9 +2,11 @@ module.exports = {
 	SUPPORTED_GAMES: ['myz', 'fbl', 'tales', 'coriolis', 'alien', 'vaesen', 't2k'],
 	SUPPORTED_LANGS: {
 		en: 'English',
+		de: 'Deutsch',
 		// fr: 'Français',
 		// sw: 'Svenska',
 	},
+	LOWERCASE_LANGS: ['en'],
```

Comment:
Why not ?
```js
Object.keys(SUPPORTED_LANGS)
```
instead of `LOWERCASE_LANGS` ?

Reply:
Maybe it is a bad name? This is because german isn't a language which is all lowercase, unlike english. So i had to distinguish this for the legend generator (see FBLLegendGenerator).

Reply:
I changed the constant to KEEP_CAPITALIZATION_LANGS (with a comment) to better represent what it does: https://github.com/Stefouch/sebedius-myz-discord-bot/pull/81/commits/1855484b2b696389836693fd49f85970b5eaefa0